test/unit/org/apache/cassandra/io/sstable/metadata/MetadataSerializerTest.java

```diff
@@ -221,73 +219,4 @@ public void testOldReadsNew(String oldV, String newV) throws IOException
         }
     }
 
-    @Test
-    public void pendingRepairCompatibility()
-    {
-        if (format == BigFormat.instance)
-        {
-            Arrays.asList("ma", "mb", "mc", "md", "me").forEach(v -> assertFalse(format.getVersion(v).hasPendingRepair()));
-            Arrays.asList("na", "nb", "nc").forEach(v -> assertTrue(format.getVersion(v).hasPendingRepair()));
-        }
-        else
-        {
-            throw Util.testMustBeImplementedForSSTableFormat();
-        }
-    }
-
-    @Test
-    public void originatingHostCompatibility()
-    {
-        if (format == BigFormat.instance)
-        {
-            Arrays.asList("ma", "mb", "mc", "md", "na").forEach(v -> assertFalse(format.getVersion(v).hasOriginatingHostId()));
-            Arrays.asList("me", "nb").forEach(v -> assertTrue(format.getVersion(v).hasOriginatingHostId()));
-        }
-        else
-        {
-            throw Util.testMustBeImplementedForSSTableFormat();
-        }
-    }
-
-    @Test
-    public void improvedMinMaxCompatibility()
-    {
-        if (format == BigFormat.instance)
-        {
-            Arrays.asList("ma", "mb", "mc", "md", "me", "na", "nb").forEach(v -> assertFalse(BigFormat.instance.getVersion(v).hasImprovedMinMax()));
-            Arrays.asList("nc", "oa").forEach(v -> assertTrue(BigFormat.instance.getVersion(v).hasImprovedMinMax()));
-        }
-        else
-        {
-            throw Util.testMustBeImplementedForSSTableFormat();
-        }
-    }
-
-    @Test
-    public void legacyMinMaxCompatiblity()
-    {
-        if (format == BigFormat.instance)
-        {
-            Arrays.asList("oa").forEach(v -> assertFalse(BigFormat.instance.getVersion(v).hasLegacyMinMax()));
-            Arrays.asList("ma", "mb", "mc", "md", "me", "na", "nb", "nc").forEach(v -> assertTrue(BigFormat.instance.getVersion(v).hasLegacyMinMax()));
-        }
-        else
-        {
-            throw Util.testMustBeImplementedForSSTableFormat();
-        }
-    }
-
-    @Test
-    public void partitionLevelDeletionPresenceMarkerCompatibility()
-    {
-        if (format == BigFormat.instance)
-        {
-            Arrays.asList("ma", "mb", "mc", "md", "me", "na", "nb").forEach(v -> assertFalse(BigFormat.instance.getVersion(v).hasPartitionLevelDeletionsPresenceMarker()));
-            Arrays.asList("nc", "oa").forEach(v -> assertTrue(BigFormat.instance.getVersion(v).hasPartitionLevelDeletionsPresenceMarker()));
-        }
-        else
-        {
-            throw Util.testMustBeImplementedForSSTableFormat();
-        }
-    }
```

Comment:
nit: Tests essentially moved to `AbstractTestVersionSupportedFeatures`?

Reply:
Yes, to separate instances for BIG and BTI formats.